Assets/MixedRealityToolkit/UX/Scripts/BoundingBoxes/BoundingBoxRig.cs

```diff
@@ -170,11 +171,13 @@ private bool ShowRig
 
         public void Activate()
         {
+            InputManager.Instance.RaiseBoundingBoxRigActivated(null, 0, gameObject);
```

Comment:
If we're not going to pass an input source or source id we can probably remove them.

Reply:
Okay, I though it was the standard way to call those methods, I'll change that.

Reply:
No worries, it would be if the bounding box itself was in input source, but its not.